Automattic/wp-calypso
# Plugin Browser: Fix announcement modal height

#### Changes proposed in this Pull Request

* Add support for custom class name for Announcement Modal block.
* Increase the height of the modal to ensure the content would fit for languages where the copy text might take more space.

**Before:**
![CleanShot 2021-12-09 at 17 12 56](https://user-images.githubusercontent.com/2722412/145426733-ed72f526-c68f-4026-b304-ef129cb53956.png)

**After:**
![CleanShot 2021-12-09 at 17 14 15](https://user-images.githubusercontent.com/2722412/145426720-76b03a30-4306-4599-925d-d14241067442.png)


#### Testing instructions

* Confirm the announcement modal copy on `/plugins` fits within the modal for all Mag-16 languages.
* If the announcement modal doesn't show for you, make sure to delete the `announcement-modal-plugins-page-revamp-XXXXX` option from the environment badge nav bar.

Fixes 348-gh-Automattic/i18n-issues
